Macros/TR10_to_StdGeos.sas

```diff
@@ -0,0 +1,63 @@
+/**************************************************************************
+ Program:  TR10_to_StdGeos.sas
+ Library:  General
+ Project:  NeighborhoodInfo DC
+ Author:   Rob Pitingolo
+ Created:  6/6/18
+ Version:  SAS 9.4
+ Environment:  Windows 7
+ 
+ Description:  This program appends NIDC standard geographies to geo2010.
+
+ Modifications:
+**************************************************************************/
+
+%macro tr10_to_stdgeos( 
+  in_ds=, 
+  out_ds=,
+  keepgeos=anc2002 anc2012 Cluster_tr2000 Cluster2017 psa2004 psa2012 VoterPre2012
+		   ward2002 ward2012 Bridgepk stantoncommons /* Keep all by default */
+);
+
+
+%macro tr_geo_corr (geo,wtname);
+
+proc sort data = general.Wt_tr10_&wtname. out = &geo.; by descending popwt; run;
+proc sort data = &geo. (keep = geo2010 &geo.) nodupkey; by geo2010; run;
```

Comment:
I don't think this does what you seem to think it will. _nodupkey_ doesn't necessarily keep the first observation in a **by** group. You need to use a **data** step to ensure that happens. See [this SAS paper](http://support.sas.com/resources/papers/proceedings12/041-2012.pdf) for an example.

Reply:
@ptatian thanks. I replaced the proc sort with a data step based on the code in that paper.